### DISCUSSION TITLE
Basal flux limited to ice shelves?

### DISCUSSION THREAD

**Discussion by mankoff** [pism/pism#532] [Q&A] [answered]
I'd like to get sub shelf basal mass change. Is this provided somewhere? Or should I derive it. It looks like I could derive it by subtracting `basal_mass_flux_grounded` from `tendency_of_ice_mass_due_to_basal_mass_flux` to get `basal_mass_flux_floating`?

**Comment by bueler** [comment, marked as answer]
At page https://www.pism.io/docs/manual/diagnostics/index.html I see that the diagnostic, spatially-variable field `basal_mass_flux_floating` is already available.  Likewise the same name gives a scalar time series; see the same page.  Do these do the job you want?  If not, let us know.

**Reply by mankoff** [reply]
Thanks. I missed this, but yes, this looks like it does what I want.
